### PR TITLE
CLOUD-674 add info about build platform

### PR DESCRIPTION
**Pull request by nmarukovich**
[CLOUD-674](https://jira.percona.com/browse/CLOUD-674) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Review by tplavcic — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| upgrade-haproxy | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| pitr | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1093/commits/a6c1ebed3e82ae7bb79f69c7f0550f82475afc11
image: `perconalab/percona-xtradb-cluster-operator:PR-1093-a6c1ebed`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |
| haproxy | passed |
| upgrade-haproxy | passed |
| recreate | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | failed |
| upgrade-proxysql | passed |
| limits | passed |
| smart-update | failed |
| demand-backup | failed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| upgrade-consistency | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| tls-issue-self | passed |
| pitr | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |
| demand-backup-encrypted-with-tls | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1093/commits/3c35c36326b8d21fec858f3ce2cbbcaf8cf60319
image: `perconalab/percona-xtradb-cluster-operator:PR-1093-3c35c363`

File: e2e-tests/README.md [hunk 1]
```diff
@@ -49,7 +49,11 @@ To specify your own repository for the Percona XtraDB Cluster Operator docker im
 ```
 export IMAGE=bob/my_repository_for_test_images:K8SPXC-622-fix-feature-X
 ```
+If you use Apple Silicon (or another ARM64 based architecture) and plan to start operator on amd64 (e.g. AWS EC2, ECS, etc.) please specify docker platform:
+```
+ export DOCKER_DEFAULT_PLATFORM=linux/amd64
```
Comment (delgod) [inline e2e-tests/README.md:54]: I think we should set env by default in the build script.
and user can change var only when non-`linux/amd64` platform needed

Comment (nmarukovich) [reply e2e-tests/README.md:54]: Could you please give a bit details why to use export is better than use docker flag? @delgod

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade-haproxy | passed |
| big-data | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| pitr | passed |
| haproxy | passed |
| scaling | passed |
| scheduled-backup | passed |
| storage | passed |
| upgrade-proxysql | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| security-context | passed |
| smart-update | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | passed |
| demand-backup-encrypted-with-tls | passed |
| init-deploy | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| limits | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| upgrade-consistency | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1093/commits/a6c1ebed3e82ae7bb79f69c7f0550f82475afc11
image: `perconalab/percona-xtradb-cluster-operator:PR-1093-a6c1ebed`

**Review by delgod — APPROVED** [review]

**Review by hors — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| recreate | passed |
| upgrade-haproxy | passed |
| scaling | passed |
| big-data | passed |
| scheduled-backup | passed |
| storage | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | passed |
| demand-backup | passed |
| smart-update | passed |
| limits | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| pitr | passed |
| one-pod | passed |
| auto-tuning | passed |
| upgrade-consistency | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| demand-backup-encrypted-with-tls | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1093/commits/12d9036fc7ef9e6215e1f83ee70edf90958ecb52
image: `perconalab/percona-xtradb-cluster-operator:PR-1093-12d9036f`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| recreate | passed |
| upgrade-haproxy | passed |
| scheduled-backup | passed |
| scaling | passed |
| storage | passed |
| big-data | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | passed |
| limits | passed |
| demand-backup | passed |
| smart-update | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| pitr | passed |
| users | passed |
| upgrade-consistency | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| demand-backup-encrypted-with-tls | failed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1093/commits/41e0af064df327ea8ad1397011362d9fb65ae08a
image: `perconalab/percona-xtradb-cluster-operator:PR-1093-41e0af06`

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |
| upgrade-haproxy | passed |
| storage | passed |
| big-data | passed |
| haproxy | passed |
| scaling | passed |
| self-healing | passed |
| init-deploy | passed |
| self-healing-chaos | passed |
| scaling-proxysql | passed |
| limits | passed |
| security-context | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| self-healing-advanced-chaos | passed |
| recreate | passed |
| affinity | passed |
| operator-self-healing | passed |
| restore-to-encrypted-cluster | passed |
| one-pod | passed |
| operator-self-healing-chaos | passed |
| demand-backup | passed |
| auto-tuning | passed |
| pitr | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| upgrade-consistency | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1093/commits/41e0af064df327ea8ad1397011362d9fb65ae08a
image: `perconalab/percona-xtradb-cluster-operator:PR-1093-41e0af06`

**Pull request merged**